apps/agreement/contracts/Agreement.sol

```diff
@@ -332,20 +332,22 @@ contract Agreement is IArbitrable, ILockManager, IAgreement, IACLOracle, AragonA
 
         uint256 currentSettingId = _getCurrentSettingId();
         uint256 lastSettingIdSigned = lastSettingSignedBy[_submitter];
-        require(lastSettingIdSigned >= currentSettingId, ERROR_SIGNER_MUST_SIGN);
+        require(lastSettingIdSigned == currentSettingId, ERROR_SIGNER_MUST_SIGN);
```

Comment:
This was already discussed here, but it was decided to leave it as it was:
https://github.com/aragon/aragon-apps/pull/1140#discussion_r430285508